packages/investor-yearn/src/api/vaults.ts

```diff
@@ -19,16 +18,5 @@ export type SupportedYearnVault = {
 
 export const getSupportedVaults = async (): Promise<SupportedYearnVault[]> => {
   const vaults = await yearnSdk.vaults.get()
-  return vaults.map((vault: Vault) => {
-    return {
-      vaultAddress: toLower(vault.address),
-      name: `${vault.name} ${vault.version}`,
-      symbol: vault.symbol,
-      tokenAddress: toLower(vault.token),
-      chain: ChainTypes.Ethereum,
-      provider: DefiProvider.Yearn,
-      type: DefiType.Vault,
-      expired: vault.metadata.depositsDisabled || bnOrZero(vault.metadata.depositLimit).lte(0)
-    }
-  })
+  return vaults.map((vault: Vault) => transformVault(vault))
```

Comment:
total nitpick but you can just do `vaults.map(transformVault)` here

Reply:
done